.pipelines/Dockerfile

```diff
@@ -1,8 +1,7 @@
 FROM ubuntu:14.04 AS build
 
 # Upgrade cmake to 3.2
-RUN apt-get update
-RUN apt-get install -y software-properties-common python-software-properties debconf-utils
+RUN apt-get update && apt-get install -y software-properties-common python-software-properties debconf-utils
```

Comment:
I don't understand very well why this change was necessary, but without it, running `docker build - < .pipelines/Dockerfile` gave me an error on this step that looked like `apt-get update` hadn't been run. Combining the two lines made it work again.

The [Docker documentation](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/) indicates that this seems to be the preferable way to run `apt-get update`.